app-backup/restic/restic-0.8.3.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit golang-vcs-snapshot bash-completion-r1
+
+DESCRIPTION="A backup program that is fast, efficient and secure"
+HOMEPAGE="https://restic.github.io/"
+SRC_URI="https://github.com/restic/restic/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+EGO_PN="github.com/restic/restic"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="test"
+
+DOCS=(
```

Comment:
Is that long list different from `doc/*.rst` (or `doc/*`?). If not, I'd suggest either declaring it inside `src_install()` or just using `dodoc` instead of `einstalldocs`. In both cases you'd be able to use the glob and avoid having to list all the files.

Reply:
Using doc/*.rst  inside DOCS doesn't work, an error is reported `restic-0.8.3.ebuild: line 20: no match: doc/*.rst`, and the doc says that the DOCS variable is `An array or space-delimited list of documentation files for the default src_install function to install using dodoc` (so, not a glob?). Using dodoc directly it will be much cleaner, thanks for the tip.

Reply:
I told you it will only work if you declare it inside `src_install` since it works relatively to current directory.

Reply:
True, I misunderstood it the first time I read it.